src/main/java/com/zendesk/maxwell/schema/MysqlSavedSchema.java

```diff
@@ -437,7 +457,8 @@ private void restoreTable(Connection connection, Database d, String name, int id
 					cRS.getString("name"), cRS.getString("charset"),
 					cRS.getString("coltype"), i++,
 					cRS.getInt("is_signed") == 1,
-					enumValues);
+					enumValues,
+					Long.valueOf(cRS.getInt("column_length")));
```

Comment:
The constructor does something like this, but I updated this part to send null it the value read was null, so then the logic is only inside the constructor.